packages/messaging/src/controllers/controller-interface.ts

```diff
@@ -66,8 +72,8 @@ export class ControllerInterface {
   async getToken(): Promise<string | null> {
     // Check with permissions
     const currentPermission = this.getNotificationPermission_();
-    if (currentPermission !== NotificationPermission.GRANTED) {
-      if (currentPermission === NotificationPermission.DENIED) {
+    if (currentPermission !== 'granted') {
+      if (currentPermission === 'denied') {
```

Comment:
why get rid of the enum?

Reply:
TS already has a type for NotificationPermission: https://github.com/Microsoft/TypeScript/blob/5f9c34bcd2933075a382abbcc8606b0a61b582fe/lib/lib.dom.d.ts#L16034

It only allows these strings to be assigned to a variable with this type, so if I made a typo in 'granted' or 'denied' here, TS would not compile.